Stardaze/OutputFormatter.swift

```diff
@@ -17,6 +23,160 @@ internal struct PrettyPrintedStringFormatter: Visitor {
         return finishedString
     }
 
+    private func makeCondensedQuery(document: Document) -> NSMutableString {
+        var query =  visit(document.queryOperation)
+
+        if let fragments = document.fragments {
+            let fragments = visit(fragments)
+
+            query.append(" ")
+            query.append(fragments)
+        }
+
+        let condensedQuery = NSMutableString(string: query)
+        condensedQuery.condenseWhitespace()
+
+        return condensedQuery
+    }
+
+    private func makeCompactDocumentString(document: Document) -> String {
```

Comment:
Yeah, I agree. It's time to clean this up in general, and I'm glad to be getting some additional eyes on it.